File: airbyte-cdk/java/airbyte-cdk/dependencies/src/testFixtures/java/io/airbyte/workers/general/DiscoverCatalogTestHarness.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.workers.general;

import io.airbyte.configoss.ConnectorJobOutput;
import io.airbyte.configoss.StandardDiscoverCatalogInput;
import io.airbyte.workers.TestHarness;

public interface DiscoverCatalogTestHarness extends TestHarness<StandardDiscoverCatalogInput, ConnectorJobOutput> {}

File: airbyte-cdk/java/airbyte-cdk/dependencies/src/testFixtures/kotlin/io/airbyte/workers/general/DiscoverCatalogTestHarness.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.workers.general

import io.airbyte.configoss.ConnectorJobOutput
import io.airbyte.configoss.StandardDiscoverCatalogInput
import io.airbyte.workers.TestHarness

interface DiscoverCatalogTestHarness :
    TestHarness<StandardDiscoverCatalogInput, ConnectorJobOutput>
